test/multipart-disk.test.js

```diff
@@ -184,6 +185,125 @@ test('should throw on file limit error', async function (t) {
   }
 })
 
+test('should throw on file save error', async function (t) {
+  t.plan(2)
+
+  const fastify = Fastify()
+  t.tearDown(fastify.close.bind(fastify))
+
+  fastify.register(require('..'))
+
+  const fileId = 'testRequestFileId'
+
+  let counter = 0
+  function toID () {
+    return fileId + counter++
+  }
+
+  fastify.post('/', async function (req, reply) {
+    t.ok(req.isMultipart())
+
+    try {
+      await req.saveRequestFiles({ toID })
+      reply.code(200).send()
+    } catch (error) {
+      reply.code(500).send()
+    }
+  })
+
+  await fastify.listen(0)
+
+  const tempFilePath = path.join(os.tmpdir(), fileId + 0 + path.extname(filePath))
```

Comment:
https://node-tap.org/docs/api/fixtures/

Reply:
Thanks for the suggestion. I used tap's tmpdir in the next commit.